src/libraries/Microsoft.Extensions.HostFactoryResolver/src/Microsoft.Extensions.HostFactoryResolver.Sources.csproj

```
@@ -9,7 +9,6 @@
     <IsSourcePackage>true</IsSourcePackage>
     <!-- This is non-shipping package. -->
     <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
```

revert the branch specific changes:

```suggestion
    <DisablePackageBaselineValidation>true</DisablePackageBaselineValidation>
    <ServicingVersion>1</ServicingVersion>
```